lib/context.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const fsPromises = require('fs').promises
+
+exports.getTestingBranchName = async function getTestingBranchName (parentPackageName) {
+  return `wiby-${parentPackageName}`
+}
+
+exports.getLocalPackageJSON = async function getLocalPackageJSON () {
+  const pkg = await fsPromises.readFile('package.json')
+  return JSON.parse(pkg)
+}
+
+/*
+The check should search both the dev and peer dependencies to find out if the dependency is
+regular, dev or peer. We only want to check on this the once :)
+ */
+exports.checkPackageInPackageJSON = function checkPackageInPackageJSON (dep, packageJSON) {
+  const dependencyKeyNames = [
+    'dependencies',
+    'devDependencies',
+    'peerDependencies',
+    'optionalDependencies'
+  ]
+  const checkKeyHasDep = (key) => Object.prototype.hasOwnProperty.call(packageJSON, key) && Object.prototype.hasOwnProperty.call(packageJSON[key], dep)
+  for (let i = 0; i < dependencyKeyNames.length; i++) {
+    if (checkKeyHasDep(dependencyKeyNames[i])) return dependencyKeyNames[i]
+  }
+  return false
```

Comment:
```suggestion
  return dependencyKeyNames.find(x => checkKeyHasDep(x))
```
loops are rarely the simpler approach :-)

Reply:
thank you for the reviews, update made.